frontend/src/Metamaps/PasteInput.js

```diff
@@ -29,12 +28,10 @@ const PasteInput = {
       if (e.dataTransfer.files.length > 0) {
         self.handleFile(e.dataTransfer.files[0], coords)
       }
-      // OMG import bookmarks 😍
+      // OMG import bookmarks 😍 (Or just text :P)
```

Comment:
But you can drag and drop just raw text from other places too, that's what I meant with this... It doesn't have to be bookmarks from my testing... (Unless i'm just confused and everything I drag and drop is actually just simulating a paste event, and it is only dragged bookmarks/links that truly drop events